migrate-profile.md

```diff
@@ -6,17 +6,17 @@ You can perform fresh install of Tanzu Application Platform 1.1 by following the
 
 ## <a id='prereqs'></a> Prerequisites
 
-Before you migrate from one Tanzu Application Platform to another:
+Before you migrate from one Tanzu Application Platform profile to another, consider the following:
 
-- For information about installing Tanzu Application Platform, see [Install your Tanzu Application Platform profile](install.md#install-profile)
-- For information about installing or updating the Tanzu CLI and plug-ins, see [Install or update the Tanzu CLI and plug-ins](install-tanzu-cli.md#cli-and-plugin)
-- For information on Tanzu Application Platform GUI considerations, see [Tanzu Application Platform GUI Considerations](tap-gui/upgrades.md#considerations)
-- Verify all packages are reconciled by running `tanzu package installed list -A`
-- Note the packages that are included in both the original profile as well as profile being migrated to. If the target profile includes additional packages, those will be added, and additional cluster resources may be needed. However, if the target profile **doesn't include** a particular component, that package will be removed as a result of the migration. If the addition or removal of the packages is of great concern, consider instead adding an additional cluster versus migrating.
+- Ensure you have the appropriate version of the Tanzu CLI
+  - For information about installing or updating the Tanzu CLI and plug-ins, see [Install or update the Tanzu CLI and plug-ins](install-tanzu-cli.md#cli-and-plugin)
+- Ensure you have a functional Tanzu Application Platform installation that is fully reconciled. 
+  - Verify all packages are reconciled by running `tanzu package installed list -A`
+- Compare the packages that are included in both the original profile as well as profile being migrated to. If the target profile includes additional packages, those will be added, and additional cluster resources may be needed. However, if the target profile **doesn't include** a particular component, that package will be **removed** as a result of the migration. If the addition or removal of the packages is of great concern, consider instead adding an additional cluster versus migrating profiles.
```

Comment:
I'd love a pointer here to wherever we tell folks to configure their values.yaml (kinda per my comment on the previous PR).